bylaws/009-per-workflow.md

```diff
@@ -0,0 +1,85 @@
+# PER Workflow
+
+A PHP Evolving Recommendation (PER) is a formal definition of best practices, utilities, reference implementations, and similar tools.  They may evolve over time as the PHP ecosystem develops.  
```

Comment:
I like the wording here, and I think it might be helpful to expand on what "ecosystem develops" means; new features added to the language, deprecations, etc. Maybe something as simple as:

> as the PHP ecosystem develops due to new PHP features and deprecations.

Reply:
Or just "as the PHP language and ecosystem develop"?

Reply:
I like that.